tests/unit/lastuser_core/test_model_user_User.py

```diff
@@ -287,6 +287,12 @@ def test_user_autocomplete(self):
         assert query_for_piglet == [piglet]
         query_for_crusoe = models.User.autocomplete(queries[2])
         assert query_for_crusoe == [crusoe]
+        query_at_character = models.User.autocomplete('@[')
+        assert query_at_character == []  # Test for empty searches
+        query_square_brackets = models.User.autocomplete('[[]]')
+        assert query_square_brackets == []
+        query_for_crusoe_with_square_brackets = models.User.autocomplete(queries[3])
+        assert query_for_crusoe_with_square_brackets == [crusoe]
```

Comment:
Assignment to variable is not required. You can simply do `assert function_call() == result`